packages/ui-components/src/features/Auth/EmailVerificationDisplay.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import Typography from '@material-ui/core/Typography';
+
+export enum STATUS {
```

Comment:
Since this is exported out of ui-components and will be imported into other contexts, could it be a more descriptive name such as EMAIL_VERIFICATION_STATUS? Or there could be a FORM_STATUS enum that is used in a bunch of places?